x/ecocredit/core/msg_mint_batch_credits.go

```diff
@@ -83,7 +83,7 @@ func validateBatchIssuances(iss []*BatchIssuance) error {
 	return nil
 }
 
-var reOriginTxNote = regexp.MustCompile(`^[a-zA-Z0-9][a-zA-Z0-9 _\-]{1,64}$`)
+var reOriginTx = regexp.MustCompile(`^[a-zA-Z0-9][a-zA-Z0-9 _\-]{1,64}$`)
```

Comment:
```suggestion
var reOriginTx = regexp.MustCompile(`^[a-zA-Z0-9][a-zA-Z0-9 _\-]{1,128}$`)
```

We should include up to 128 chars as the following would be a valid verra serial number:
`0001-999999999999999-999999999999999-VCS-VCU-003-VER-US-0003-01012020-31122020-1`

Reply:
From here: https://verra.org/wp-content/uploads/2020/09/VCU-Serial-Number-Help-Format.pdf